.github/workflows/test.yml

```diff
@@ -15,6 +15,9 @@ jobs:
         - { python-version: "3.7", os: "ubuntu-latest", backend-db: sqlite }
         - { python-version: "3.8", os: "ubuntu-latest", backend-db: sqlite }
         - { python-version: "3.9", os: "ubuntu-latest", backend-db: sqlite }
+        - { python-version: "3.7", os: "windows-latest", backend-db: sqlite }
+        - { python-version: "3.8", os: "windows-latest", backend-db: sqlite }
+        - { python-version: "3.9", os: "windows-latest", backend-db: sqlite }
```

Comment:
👍 hopefully we can start testing Windows + Postgres at some point

Reply:
https://github.com/meltano/meltano/issues/6281 :D yes, I'll add this issue link inline as well